src/realsense.def

```diff
@@ -132,6 +135,13 @@ EXPORTS
 
     rs2_log_to_console
     rs2_log_to_file
+    rs2_log_to_callback
+    rs2_log_to_callback_cpp
+    
+    rs2_build_log_message
+    rs2_get_log_message_line_number
+    rs2_get_log_message_filename
```

Comment:
What about get log severity level?

Reply:
It's always delivered to the callback alongside the message, as an argument